### PR TITLE
Periodically refresh ConversationsList.

### PR DESCRIPTION
**Pull request by ma12-co**
Signed-off-by: Marco Ambrosini <marcoambrosini@pm.me>

### CODE REVIEW

File: src/components/Navigation/ConversationsList/ConversationsList.vue [hunk 1]
```diff
@@ -45,20 +45,29 @@ export default {
...
 			this.joinConversation(selectedConversationToken)
 			this.$router.push({ path: `/call/${selectedConversationToken}` })
+		},
+		async fetchConversations() {
+			/** Fetches the conversations from the server and then adds them one by one
+			 * to the store.
+			 */
+			const conversations = await fetchConversations()
+			conversations.data.ocs.data.forEach(conversation => {
+				this.$store.dispatch('addConversation', conversation)
```
Comment (nickvergessen) [inline src/components/Navigation/ConversationsList/ConversationsList.vue]: This means when someone else deletes a conversation it's not going to be removed, right?

File: src/components/Navigation/ConversationsList/ConversationsList.vue [hunk 2]
```diff
@@ -45,20 +45,30 @@ export default {
...
-	async beforeMount() {
-		/** Fetches the conversations from the server and then
-		 * adds them one by one to the store.
-		 */
-		const conversations = await fetchConversations()
-		conversations.data.ocs.data.forEach(conversation => {
-			this.$store.dispatch('addConversation', conversation)
-		})
+	beforeMount() {
+		this.fetchConversations()
```
Comment (skjnldsv) [inline src/components/Navigation/ConversationsList/ConversationsList.vue]: What would be best is to have an initialstate for this!
Would be much more snappier! ;)

Comment (ma12-co) [reply src/components/Navigation/ConversationsList/ConversationsList.vue]: What do you mean??

Comment (nickvergessen) [reply src/components/Navigation/ConversationsList/ConversationsList.vue]: Something for later

File: src/store/conversations.js [hunk 3]
```diff
@@ -21,6 +21,15 @@
  */
 import Vue from 'vue'
 
+const getDefaultState = () => {
+	return {
+		conversations: {
+		},
+		conversationsNames: {
```
Comment (skjnldsv) [inline src/store/conversations.js]: What would be the data inside this?
Using token as indexes?
What is the purpose of this conversationsNames?

Comment (ma12-co) [reply src/store/conversations.js]: Ehrm, currenly nothing

**Pull request merged**